tasks/3.test.js

```diff
@@ -0,0 +1,20 @@
+const { motto, mottoFilter } = require("./3");
+
+describe("Return motto by house name", () => {
+  describe("motto", () => {
+    test("Should return Our Blades Are Sharp", () => {
```

Comment:
Usually we try to state the expected result given a certain condition in the test name, for example:
`test('should return "Our Blades Are Sharp" motto given a "Bolton" name')`